File: packages/implementations/OneLang-Reflect-v0.1/CSharp/src/ReflectedValue.cs
using One.Ast;

public class ReflectedValue
{
    public IType getDeclaredType() { return null; }
    public string getStringValue() { return null; }
    public bool getBooleanValue() { return false; }
    public ReflectedValue[] getArrayItems() { return null; }
    public string[] getMapKeys() { return null; }
    public object getUniqueIdentifier() { return null; }
    public IType getValueType() { return null; }
    public ReflectedValue getField(string name) { return null; }
    public string getEnumValueAsString() { return null; }
    public bool isNull() { return false; }
    public ReflectedValue getMapValue(string key) { return null; }
}

File: packages/implementations/OneLang-Reflect-v0.1/Java/src/ReflectedValue.java
package io.onelang.std.reflect;

import OneLang.One.Ast.Interfaces.IType;

public class ReflectedValue
{
    public IType getDeclaredType() { return null; }
    public String getStringValue() { return null; }
    public Boolean getBooleanValue() { return false; }
    public ReflectedValue[] getArrayItems() { return null; }
    public String[] getMapKeys() { return null; }
    public Object getUniqueIdentifier() { return null; }
    public IType getValueType() { return null; }
    public ReflectedValue getField(String name) { return null; }
    public String getEnumValueAsString() { return null; }
    public Boolean isNull() { return false; }
    public ReflectedValue getMapValue(String key) { return null; }
}